packages/@eventual/aws-runtime/src/stores/entity-store.ts

```diff
@@ -437,15 +444,21 @@ export class AWSEntityStore extends EntityStore {
 
     return {
       Key: this.entityKey(key),
-      UpdateExpression: [
-        "SET #__version=if_not_exists(#__version, :__startingVersion) + :__versionIncrement",
-        ...Object.keys(valueRecord).map(
-          (key) =>
-            `${formatAttributeNameMapKey(key)}=${formatAttributeValueMapKey(
-              key
-            )}`
-        ),
-      ].join(","),
+      UpdateExpression:
+        [
+          "SET #__version=if_not_exists(#__version, :__startingVersion) + :__versionIncrement",
+          ...Object.keys(valueRecord).map(
+            (key) =>
+              `${formatAttributeNameMapKey(key)}=${formatAttributeValueMapKey(
+                key
+              )}`
+          ),
```

Comment:
These could be undefined?

Reply:
Should you also check if the value is undefined?

Reply:
thats fine, `{ S: undefined }` will also remove